# SwitchState.srv
---
bool[] states
bool success
string message
